_data/projects.yml

```diff
@@ -97,82 +98,60 @@ projects</a>, including <a href=\"https://www.vmware.com/\">VMware</a>,
-demo video</a> is available."
-
+program. Also in 2018, The Uptane Alliance was officially formed as a nonprofit group
+under the auspices of IEEE/ISTO to oversee standardization of the technology's design and implementation."
+      in progress: "A working group has prepared an initial version of the <a href=\"https://uptane.github.io/uptane-standard/uptane-standard.html\">Standards document</a>."
       people:
         - *trishank_kuppusamy
         - *sebastien_awwad
-        - *vlad_diaz
         - *marina_moore
         - *lois_delong
-        - name: Damon McCoy
-          link: "http://engineering.nyu.edu/people/damon-mccoy"
-        - name: Russ Bielawski (UMTRI)
-          link: "http://umtri.umich.edu/who-we-are/research-groups/engineering-systems"
-        - name: Cameron Mott (SWRI)
-          link:
-        - name: John Liming (SWRI)
-          link:
-        - name: Sam Lauzon (UMTRI)
-          link: "http://umtri.umich.edu/who-we-are/research-groups/engineering-systems"
-        - name: André Weimerskirch (UMTRI)
-          link: "http://isd.engin.umich.edu/people/andre-weimerskirch"
-        - *justin_cappos
+        - name: André Weimerskirch (Uptane Alliance Board)
+        - name: Ira McDonald (Uptane Alliance Board)
+        - *justin_cappos (Uptane Alliance Board)
       tags:
         - *software_updates
         - *security
         - *internet_of_things
 
-
   - in-toto: &in-toto
       name: "in-toto"
       anchor: "in-toto"
       image: "img/projects/in-toto_diagram.png"
       status: *starting
       site: "https://in-toto.io"
-      description: "Do you know who has handled your software prior to its installation on your machine? Even if developers are careful to secure each step in their products' supply chain, there is little assurance about what happens in-between these steps. The in-toto system holistically enforces the integrity of a software supply chain by gathering and signing information about each step in the process. As such, in-toto provides accountability about how software is written, packaged and distributed...and by who."
-      products: "The in-toto software has already been integrated into several open source
-    projects. Most recently, Arch Linux announced that a patch to check for
-    invalid tags in git will be integrated into the next release of its <a href=\"https://git.archlinux.org/pacman.git/commit/?id=39319c1860d200a9b4a3cc2c6975e3cece502f2d\">pacman</a>
-    utility. We welcome you to download the in-toto
-    instructions, which includes a <a href=\"https://github.com/in-toto/in-toto/tree/develop/demo/\">demo version</a> of our software,
-    or to clone our <a href=\"https://github.com/in-toto/in-toto/\">repository</a> and follow the
-    directions to integrate in-toto into your software project!"
+      description: "Even if developers are careful to secure each step in
```

Comment:
I think it would be stronger to talk about more of the problem here.  It sort of seems like we're really focused on only trying to protect in between steps in the supply chain.  I think we can make the point stronger that it is end-to-end security, including in between existing steps.

Reply:
Fixed